EvidenceApi/V1/UseCase/CreateEvidenceRequestUseCase.cs

```diff
@@ -98,5 +108,29 @@ private DeliveryMethod ParseDeliveryMethod(string deliveryMethod)
         {
             return Enum.Parse<DeliveryMethod>(deliveryMethod, true);
         }
+
+        private static void ThrowException(List<DeliveryMethod> deliveryMethods, List<DeliveryMethod> exceptions)
+        {
+            var successfulDeliveryMethods = deliveryMethods.Except(exceptions).ToList();
+
+            if (exceptions.Count != 0)
+            {
+                if (deliveryMethods.Count > 1)
+                {
+                    throw new NotificationException(
+                        "There was an error sending the request by " +
+                        String.Join(", ", exceptions).ToLower() +
+                        ". The request was still sent by " +
```

Comment:
I don't think we should include which delivery methods were successful in this exception message - the message should only tell callers of the API which methods failed